src/Symfony/Component/Notifier/Bridge/OneSignal/OneSignalOptions.php

```diff
@@ -108,6 +108,13 @@ public function recipient(string $id): static
         return $this;
     }
 
+    public function isExternalUserId(bool $flag = true): static
```

Comment:
Can you please add a comment to the method with a link to the corresponding docs? Thanks

Reply:
OneSignal docs or Symfony docs?

Reply:
OneSignal docs please, sorry for being not clear

Reply:
Let me know if that's what you had in mind

Reply:
Perfect, thanks